src/test/java/org/apache/datasketches/kll/KllDirectDoublesSketchTest.java

```diff
@@ -644,6 +645,25 @@ public void checkMergeExceptionsWrongType() {
     try { sk2.merge(sk1); fail(); } catch (ClassCastException e) { }
   }
 
+  @Test
+  public void checkVectorUpdate() {
+    WritableMemory dstMem = WritableMemory.allocate(6000);
```

Comment:
do we care about "final" in tests?

Reply:
I will fix for this file, but removing these in all the tests will be a cleanup task for later.